src/cephadm/cephadm.py

```diff
@@ -6400,6 +6550,14 @@ def _dispatch_deploy(
                       config=config, keyring=keyring,
                       deployment_type=deployment_type,
                       ports=daemon_ports)
+    elif daemon_type == CephNvmeof.daemon_type:
+        config, keyring = get_config_and_keyring(ctx)
+        uid, gid = 65534, 65534  # TODO: check this
```

Comment:
we can't use the `extract_uid_gid` here? I guess that relies on the container being based on the ceph container which I guess the nvme-of container is not? Also, is using the `nobody` user what was recommended by the team creating the nvme-of container?

Reply:
Ideally we should use `167:167` (`ceph:ceph`), but that's not yet possible, since mounting hugepages requires superuser rights...

Reply:
@adk3798 @epuertat `167:167` seems to work fine (container get into running state and I can run the `get_systems` cmd). I changed the code to use `ceph:ceph` (hardcoded by now as the base image of the container is not the ceph image).